### PR TITLE
Feature/add eco places and shopping list

### PR DESCRIPTION
**Pull request by ihordubas**
Create eco places and shopping list for user profile

### CODE REVIEW

File: src/app/component/user/profile/eco-places/eco-places.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="eco-places-content">
+    <div class="header-position">
+        <h2 class="header">My eco places <span>{{ecoPlaces.length}}</span></h2>
+        <a class="btn-see-all">See all</a>
+    </div>
+    
+    <ul class="eco-place-list">
+        <li class="item" *ngFor="let ecoPlace of showEcoPlaces">{{ecoPlace}}</li>
```
Comment (nvolodymyr) [inline src/app/component/user/profile/eco-places/eco-places.component.html:8]: align in the column

File: src/app/component/user/profile/eco-places/eco-places.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (nvolodymyr) [inline src/app/component/user/profile/eco-places/eco-places.component.spec.ts:2]: del gap

File: src/app/component/user/profile/shopping-list/shopping-list.component.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="shopping-list-content">
+    <div class="header-position">
+        <h2 class="header">Shopping list <span>{{shoppingList.length}}</span></h2>
+        <a class="btn-see-all">See all</a>
+    </div>
+    
+    <ul class="shopping-list">
+        <li class="item" *ngFor="let spoppingItem of showShoppingList">{{spoppingItem}}
```
Comment (nvolodymyr) [inline src/app/component/user/profile/shopping-list/shopping-list.component.html:8]: align in the column

File: src/app/component/user/profile/shopping-list/shopping-list.component.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+            <mat-checkbox class="checkbox" color="primary"></mat-checkbox>
```
Comment (nvolodymyr) [inline src/app/component/user/profile/shopping-list/shopping-list.component.html:9]: the same

File: src/app/component/user/profile/eco-places/eco-places.component.html (hunk 1 above)
Comment (ihordubas) [reply src/app/component/user/profile/eco-places/eco-places.component.html:7]: fixed

File: src/app/component/user/profile/eco-places/eco-places.component.spec.ts (hunk 2 above)
Comment (ihordubas) [reply src/app/component/user/profile/eco-places/eco-places.component.spec.ts]: fixed

File: src/app/component/user/profile/shopping-list/shopping-list.component.html (hunk 3 above)
Comment (ihordubas) [reply src/app/component/user/profile/shopping-list/shopping-list.component.html:7]: fixed

File: src/app/component/user/profile/shopping-list/shopping-list.component.html (hunk 4 above)
Comment (ihordubas) [reply src/app/component/user/profile/shopping-list/shopping-list.component.html:8]: fixed

**Pull request merged**